src/dvc_objects/db.py

```diff
@@ -59,6 +60,19 @@ def get(self, oid: str) -> Object:
             oid,
         )
 
+    def add_bytes(self, oid: str, data: Union[bytes, BinaryIO]) -> None:
```

Comment:
I changed this to `add_bytes`, but I still find `write` to be a better naming than `add`. `write` has it's counterpart `read`, what does `add` have? Also, `add` adds a `file` to the database, this one _writes_/_sets_ it to the database.

Reply:
You can think of `add` as a higher-level API that can theoretically use `write` API, so I don't think there needs to be a symmetry as they are on different levels. :)